kasa/cli/main.py

```diff
@@ -81,6 +81,7 @@ def _legacy_type_to_class(_type):
         "led": "device",
         "alias": "device",
         "reboot": "device",
+        "factory_reset": "device",
```

Comment:
I noticed that the command is called `factory-reset` in the device group, but needs to be called as `factory_reset` for the main command. Unsure if this is expected behavior.

Reply:
I think that is expected behaviour as it's a python import rather than going through the click logic.  However are we sure we want to expose this as a top level shortcut? The top level shortcuts were really for backwards compatibility and common commands, perhaps this should just live under the `device` command.

Reply:
Yeah, that sounds reasonable, let's remove it!